yo/blockchain_follower.py

```diff
@@ -55,7 +55,17 @@ class YoBlockchainFollower(YoBaseService):
                 return op
        return None
    async def async_ops(self,loop,b):
-       ops = b.stream_from(start_block=int(self.yo_app.config.config_data['blockchain_follower']['start_block']) )
+       start_block = str(self.yo_app.config.config_data['blockchain_follower'].get('start_block',''))
```

Comment:
You're awesome @GarethNelson, but this solution is too awesome and complicated. Pick the best sane default and implement that.